graphix/__init__.py

```diff
@@ -1,7 +1,9 @@
 """Optimize and simulate measurement-based quantum computation."""
 
 from graphix.generator import generate_from_graph
-from graphix.graphsim import GraphState
+from graphix.graphsim.graphstate import GraphState
```

Comment:
Could you clarify the rationale behind this change? I would actually prefer to keep `graphix.graphsim` as an abstraction layer that hides the exact layout of its submodules.

Reply:
Two reasons.

1. `mypy` warning

It was marked as invalid by `mypy` until `__all__` attributes are added in this PR.

2. `graphix.graphsim.graphstate` is not hidden

If we really strive to use `graphix.graphsim` as an abstraction layer, we should rename `.graphstate` to `._graphstate` (private module).
I would like to leave it as-is since it is out of the scope of this PR.

Reply:
Ok, thanks!